.grit/patterns/index_of_to_includes.md

```diff
@@ -39,15 +54,15 @@ foo.indexOf("a") != -1;
 ```typescript
 !foo.includes("a");
 
-!foo.includes("a");
+!foo.includes("a")
```

Comment:
From my understanding Marzano doesn't include formatting like Scala did, so it seems like the proper separation of concerns to just leave this as is and save the formatting for a format step in an actual migration.